scripts/combatdata.gd

```diff
@@ -116,42 +116,49 @@ var enemygrouppools = {
 		description = 'You spot a [color=aqua]group of cave goblins[/color] moving through the tunnels. ',
 		descriptionambush = "A [color=aqua]group of cave goblins[/color] jumps on you. ",
 	},
-	dryad = {
-		units = [['dryad',1,1]], awareness = 0, captured = null, special = null,
-		description = 'You spot a wild [color=aqua]$race $child[/color] walking through the woods.',
-	},
+	dryad2 = {
```

Comment:
the original "dryad" group is missing here but still listed in explorationregions.gd. this is an error

Reply:
Fixed